styles.css

```diff
@@ -388,10 +395,365 @@
   }
 
   label {
-    font-family: 'Poppins', sans-serif;
+    font-family: "Poppins", sans-serif;
     font-size: 10px;
     line-height: 15px;
     font-weight: 400;
     color: #7d7c84;
   }
+
+  .grid-picture {
+    display: none;
+  }
+
+  .image6 {
+    display: none;
+  }
+
+  .image7 {
+    display: none;
+  }
+
+  .work-desktop {
+    display: none;
+  }
+
+  .Project-Desktop {
+    display: none;
+  }
+}
+
+/* Desktop version */
+
+@media (min-width: 768px) {
+  #header {
+    border-bottom: 1px solid #dddfe6;
+    height: 6.25rem;
+  }
+
+  .Menu {
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    height: 6.25rem;
+  }
+
+  .portfolio-links {
+    list-style: none;
+    display: flex;
+  }
+
+  .Menu-line {
+    display: none;
+  }
+
+  .nav-link a {
+    text-decoration: none;
+    padding: 1rem;
+    color: #1a2236;
+  }
+
+  .image1 {
+    display: none;
+  }
+
+  .image2 {
+    display: none;
+  }
+
+  .image3 {
+    display: none;
+  }
+
+  .image4 {
+    display: none;
+  }
+
+  .image5 {
+    display: none;
+  }
+
+  .Project-Mobile {
+    display: none;
+  }
+
+  #headline {
+    display: flex;
+  }
+
+  .Intro {
+    width: 50%;
+    position: relative;
+    height: 100%;
+    padding-top: 200px;
+  }
+
+  .headline-pic {
+    position: relative;
+    width: 50%;
+    background-image: url(images/Thur.png);
+    height: 54.813rem;
+    margin-bottom: 13rem;
+  }
+
+  h1 {
+    font-size: 110px;
+    line-height: 110px;
+  }
+
+  .Collaboration {
+    width: 15.875rem;
+  }
+
+  /* Desktop works */
+  .Project-Desktop {
+    display: block;
+  }
+
+  .Projects {
+    position: relative;
+    display: inline-block;
+    font-size: 110px;
+    line-height: 110px;
+    transform: rotate(-90deg);
+  }
```

Comment:
- [ ] The margins in this section are different from the Figma template, kindly make sure to stick to the design as much as possible.

![rev2](https://user-images.githubusercontent.com/53347966/230517524-70ae8d4c-a9a0-404c-ae5f-98ddcb1d19c6.JPG)